src/plugins/interactive_setup/server/plugin.ts

```diff
@@ -57,6 +56,16 @@ export class UserSetupPlugin implements PrebootPlugin {
       return;
     }
 
+    const configPath = this.initializerContext.env.mode.dev
+      ? this.initializerContext.env.configs.find((config) => config.endsWith('.dev.yml'))
+      : this.initializerContext.env.configs[0];
+    if (!configPath) {
+      this.#logger.debug(
+        'Interactive setup mode will not be activated since Kibana configuration file is not available.'
+      );
```

Comment:
Is this even possible? Should we just write to `kibana.yml` file if it doesn't exist?

Reply:
I'm not sure if we want to write to `kibana.yml` in dev when developer doesn't use dev config?

Reply:
Why not? If it's the config file they use for dev I think we should write to it.

Reply:
> Is this even possible? 

Checked the code, and it seems it's not possible indeed. I hoped we supported an pure ENV variable based setup, but it's not the case :cry: 

> Why not? If it's the config file they use for dev I think we should write to it.

That's a fair point, let's try this and see how it goes.